app/models/oidc_configuration.rb

```diff
@@ -55,7 +55,7 @@ def attributes
     end
   end
 
-  belongs_to :oidc_configurable, polymorphic: true
+  belongs_to :oidc_configurable, polymorphic: true, inverse_of: :oidc_configuration
```

Comment:
I noticed that Rails didn't figure out the reverse association here, so helping it do.